File: MY PROGRAMS/LB/CPP/SELECTION,SEQUENCE AND  ITERATION/a7q5.cpp
/* Write a program which accept N and print first 5 multiples of N.

Input:	4
Output: 4 8 12 16 20

*/




#include<iostream>

using namespace std;

class Multiples
{
	public:
		int i=0,imult=1;

	public:
		void Display(int ino)
		{
			if(ino<0)
			{
				ino=-ino;
			}

			for(i=1;i<=5;i++)
			{
				imult=ino*i;

				cout<<imult<<" ";
			}
		}
};

int main()
{
	int ivalue=0;

	cout<<"Enter the no.:";
	cin>>ivalue;

	Multiples mobj;

	mobj.Display(ivalue);

	return(0);
}

File: MY PROGRAMS/LB/JAVA/NUMBER/a7q5.java
/*	5. Write a program which accept N and print first 5 multiples of N.

	Input:	4
	Output:	4 8 12 16 20	*/




import java.util.*;

class Assignment7
{
	public void Display(int ino)
	{
		int i=0,imult=1;

		if(ino<0)
		{
			ino=-ino;
		}

		if(ino==0)
		{
			return;
		}

		for(i=1;i<=5;i++)
		{
			imult=ino*i;

			System.out.print(imult+"\t");
		}
	}
}

class a7q5
{
	public static void main(String arg[])
	{
		Scanner sobj=new Scanner(System.in);

		int ivalue=0;

		System.out.println("Enter the no.: ");
		ivalue=sobj.nextInt();

		Assignment7 aobj=new Assignment7();

		aobj.Display(ivalue);
	}
}
